grafana/tasks/install.yml

```diff
@@ -1,11 +1,12 @@
 ---
 - name: Download Grafana
+  when: grafana_download | default(true)
```

Comment:
It's safe to remove the `| default(true)` since a default value is set in `defaults/main.yml` (ditto for the influxdb one)